docusaurus.config.js

```diff
@@ -66,6 +66,7 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      image: 'img/banner.png',
```

Comment:
@nirutthyu, can you please refer to this https://docusaurus.io/docs/2.x/seo, I believe the meta image should be under the `metadata` config object,

You can check your changes here https://metatags.io/?url=https%3A%2F%2Fdeploy-preview-78--react-play-docs.netlify.app. The meta description shows `banner` instead of the actual one we have.

<img width="1170" alt="image" src="https://github.com/user-attachments/assets/93b7c1f0-41bf-468f-b71d-bfde00803165">